ang/crmMonaco.js

```diff
@@ -65,6 +65,12 @@
             // FIXME: else: retry?
           };
 
+          if ($attr.ngDisabled) {
```

Comment:
@colemanw I think this change to crmMonaco is the only part of this pr that needs review (the rest is internal to the extension & gets a pass under 'alpha/hidden' level ) 

do you have any thoughts on it?

Reply:
FWIW, there are similar techniques in `ui-bootstrap-tpls.js` which inspired this.

Reply:
@totten ok - is getting this merged holding you up? I am not familiar with that code but I guess if it's a pre-existing pattern....

Reply:
I guess my feeling is that if you feel confident in that change then go ahead & merge this